arroyo/processing/strategies/dead_letter_queue/policies/abstract.py

```diff
@@ -1,5 +1,19 @@
 from abc import ABC, abstractmethod
-from typing import Any, Sequence
+from dataclasses import dataclass
+from datetime import datetime
+from typing import Optional, Sequence, Union
+
+Serializable = Union[str, bytes]
+
+
+@dataclass(frozen=True)
+class InvalidMessage:
```

Comment:
This seems conflicting with the implementation you provided here https://github.com/getsentry/arroyo/pull/57/files#diff-7ca52af34de9ab8fed8d220da046d4023236fcdbadc3a979718477f289d04713R11

Which one is the most up to date one ?

Reply:
The other one is most up to date, I just cloned the bare minimum here to be "consistent" and can deal with merge conflicts.

Maybe I should pull out this class/concept into its own PR and have it merged first.

Reply:
https://github.com/getsentry/arroyo/pull/64